packages/esm-active-visits-app/src/active-visits-widget/active-visits.resource.tsx

```diff
@@ -1,7 +1,10 @@
 import useSWR from 'swr';
 import dayjs from 'dayjs';
+import isToday from 'dayjs/plugin/isToday';
 import { openmrsFetch, Visit, useSession } from '@openmrs/esm-framework';
 
+dayjs.extend(isToday);
```

Comment:
Added this because it was failing the lint check locally. From what I can see, this extend call isn't made elsewhere yet.